### DISCUSSION TITLE
Notion integrate some notion database ?

### DISCUSSION THREAD

**Discussion by TacioRibeiro** [FlowiseAI/Flowise#667] [Q&A]
Hello guys, does someone know how to embed a Notion Database in a flowise workflow, for the chatbot tells me some information about the Notion Database ?

![Capture d'écran 2023-08-01 004904](https://github.com/FlowiseAI/Flowise/assets/139287430/1a0acc08-e559-487a-887e-741d8473efa6)

**Comment by Anaumer** [comment]
Hi, 
it looks like your problem is based in the pinecone node. Have you pasted in your environment and the name of your index?

I'm unfortunately stuck as well. Neither a Notion Page nor a Database as an Input is working. 
The response is always that there is no information about that topic, which was asked (and of course is related to the database). 

Anyone any ideas?

**Comment by chungyau97** [comment]
Hi @TacioRibeiro,

You will need to follow [this](https://developers.notion.com/docs/create-a-notion-integration#before-you-begin) step-by-step guide to access your Notion's Database.
You can learn more using the information logo, shown below:
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/e0eb1894-26e8-4818-a686-0e7fa244dda7)
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/5d896d2c-7d47-4d6e-b0a6-b0a2f618ca99)
